package/src/memmon.cpp

```diff
@@ -75,3 +75,9 @@ std::map<std::string, double> const memmon::get_json_average_stats(
     unsigned long long elapsed_clock_ticks) {
   return mem_average_stats;
 }
+
+// Collect related hardware information
+std::map<std::string, std::map<std::string, std::string>> const memmon::get_hardware_info() {
+  std::map<std::string, std::map<std::string, std::string>> result{};
+  return result; 
```

Comment:
It would be nice here to parse `/proc/meminfo` and get `MemTotal`

Reply:
Good point, now added.